iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/DanglingLineImpl.java

```diff
@@ -31,15 +239,20 @@
 
     private final String ucteXnodeCode;
 
+    private GenerationImpl generation;
+
     private CurrentLimitsImpl limits;
 
+    private final ReactiveLimitsHolderImpl reactiveLimits;
```

Comment:
Maybe the reactiveLimits should be hold by the generation object: does it make sense to have no generation but a reactiveLimits?

Reply:
No, you are right, I am changing it